### PR TITLE
feat: add upcoming events carousel in the venue page

### PR DESCRIPTION
**Pull request by nikomakela**
LIIKUNTA-319

There are some related events:

http://localhost:3000/fi/paikat/tprek:41835

image

image

There are no related events:

http://localhost:3000/fi/paikat/tprek:40419

image

image

### CODE REVIEW

**Comment by nikomakela** [comment]
The label should be fixed 

image

- [ ] Fix the label

File: apps/sports-helsinki/src/domain/venue/upcomingEvents/upcomingEvents.module.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.upcomingEvents {
+  --padding-bottom: 8em;
+  padding: var(--spacing-xl) 0;
+  padding-bottom: var(--padding-bottom);
+  //   background: var(--color-black-5);
```
Comment (kristianjokela) [inline apps/sports-helsinki/src/domain/venue/upcomingEvents/upcomingEvents.module.scss:5]: commented out code - could be deleted too.

**Review by kristianjokela — APPROVED** [review]

**Comment by nikomakela** [comment]
> * [x]  Fix the label
> * [x]  Fix the empty location

image

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr173.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr173.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr173.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://harrastukset-pr173.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr173.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by nikomakela** [comment]
I'm pretty sure this is mergeable at this point, eventhough all the CI is failing. Azure have had issues today. But, this PR can wait for better times.

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr173.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr173.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr173.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr173.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr173.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr173.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr173.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr173.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**